tests/pull.bats

```diff
@@ -266,3 +266,49 @@ load helpers
   iid=$output
   run_buildah rmi ${iid}
 }
+
+@test "pull-policy" {
+  mkdir ${TESTDIR}/buildahtest
+  run_buildah 125 pull --signature-policy ${TESTSDIR}/policy.json --policy bogus alpine
+  expect_output --substring "unrecognized pull policy bogus"
+
+  #  If image does not exist the never will fail
+  run_buildah 125 pull --signature-policy ${TESTSDIR}/policy.json --policy never alpine
+  expect_output --substring "no such image"
+  run_buildah 125 inspect alpine
+  expect_output --substring "image not known"
+
+  # create bogus alpine image
+  run_buildah from --signature-policy ${TESTSDIR}/policy.json scratch
+  cid=$output
+  run_buildah commit -q $cid docker.io/library/alpine
+  iid=$output
+
+  #  If image does not exist the never will succeed, but iid should not change
+  run_buildah pull --signature-policy ${TESTSDIR}/policy.json --policy never alpine
+  expect_output $iid
+
+  # Pull image by default should change the image id
+  run_buildah pull --policy always --signature-policy ${TESTSDIR}/policy.json alpine
+  if [[ $output == $iid ]]; then
```

Comment:
I'm not sure this tests what you think it tests. `$output` includes stderr, so on a successful pull it will include progress messages. I think it might be better to do `if [[ $output ~ $iid ]]; then`. But I'd need to pull & build & test on my end to make sure, right now this is just a quick first pass.

Reply:
Added -q which should get rid of the progress.  I would rather do exact match just in case.